src/api/l_filesystem.c

```diff
@@ -484,6 +485,18 @@ static int libLoader(lua_State* L) {
   return 1;
 }
 
+static int libLoader(lua_State* L) {
+  const char* module = lua_tostring(L, 1);
+  const bool allInOneFlag = false;
+  return(libLoaderCommon(L, allInOneFlag));
+}
+
+static int libLoaderAllInOne(lua_State* L) {
+  const char* module = lua_tostring(L, 1);
+  const bool allInOneFlag = true;
+  return(libLoaderCommon(L, allInOneFlag));
```

Comment:
Small requests:

- Could `allInOneFlag` just be named `allInOne`?
- `const bool` should just be `bool` (There's nothing _wrong_ with const there, it is just a little unconventional for the codebase style)
- Please remove parentheses around returns: `return libLoaderCommon(...);`)

Reply:
1. Yes. But IMHO `allInOneFlag` more suitable, because it just switches function behavior and don't provide any additional information.
2. OK
3. OK